### DISCUSSION TITLE
Is there a listing userguide anywhere to the Pages _config.yml file?

### DISCUSSION THREAD

**Discussion by DavidThielen** [community/community#72748] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I can find various answers about how to accomplish specific things in the _config.yml file. But no general guide.

Is there a user guide to the _config.yml file? And is there a guide to creating a Pages theme? I asked GitHub support if there is and they told me they don't answer that question.

So do we all have to guess and work off the specific Q&A we can find on it?

thanks - dave

**Comment by bartzalewski** [comment, marked as answer]
Maybe these will help

https://jekyllrb.com/docs/configuration/
https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll

**Reply by DavidThielen** [reply]
Again, thank you

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
